Sources/StreamChat/APIClient/Endpoints/ChannelEndpoints.swift

```diff
@@ -67,6 +67,26 @@ extension Endpoint {
         )
     }
 
+    static func acceptInvite(cid: ChannelId, userId: String, message: String?) -> Endpoint<EmptyResponse> {
```

Comment:
the `userId` parameter is not needed, only server-side API clients use that

Reply:
Okay, I saw it in the android code, I was wondering why it is needed